static/js/modules/filings.js

```diff
@@ -34,6 +34,10 @@ var renderModal = tables.modalRenderFactory(
       var result = response.results.length ?
         response.results[0] :
         {};
+
+      result.amendment_version = helpers.amendmentVersion(result.most_recent);
```

Comment:
I think you need this same thing on the reports pages. It's missing from the panels.